www/pfSense-pkg-squid/files/usr/local/pkg/squid_auth.xml

```diff
@@ -141,6 +141,22 @@
 			<size>5</size>
 			<default_value>5</default_value>
 		</field>
+		<field>
+			<fielddescr>Authentication Max User IP</fielddescr>
+			<fieldname>max_user_ip</fieldname>
+			<description>
+				<![CDATA[
+				This will be matched when the user attempts to log in from more than <number> different IP addresses.<br/>
+				Browsing from any further IP addresses is denying until the Authentication TTL has expired.<br/>
+				<span class="text-info">Default: none</span>
+				<div class="infoblock">
+				<span class="text-info">Note:</span> In acceleration mode or where there is mesh of child proxies, clients may appear to come from multiple addresses if they are going through proxy farms, so a limit of 1 may cause user problems.
```

Comment:
This seems a bit confusing in the wording. Perhaps:

> Acceleration mode or multiple child proxies in a mesh or farm may cause clients to appear as though they are using multiple IP addresses. A limit of 1 may cause problems for users in these cases.

Reply:
Ok, fixed